### DISCUSSION TITLE
Websocket not connecting

### DISCUSSION THREAD

**Discussion by bhushankumar-developer** [supabase/supabase#951] [Questions] [answered]
https://github.com/supabase/realtime/tree/master/examples/node-js

I have tried this example step by step as per README.

```
1. docker-compose up
2. npm start
```

But i have faced issue of refused connection to websocket as per the realtime docker logs.

```
realtime_1_f0e89c4cdc41 | 2021-03-26 09:47:16.891 [info] REFUSED CONNECTION TO RealtimeWeb.UserSocket in 118µs
realtime_1_f0e89c4cdc41 |   Transport: :websocket
realtime_1_f0e89c4cdc41 |   Serializer: Phoenix.Socket.V1.JSONSerializer
realtime_1_f0e89c4cdc41 |   Parameters: %{"vsn" => "1.0.0"}
```

Do i have to additional steps for it, How can i fix this?

**Comment by sumitalp** [comment, marked as answer]
I'm facing the same problem both from `node-js` and `next-js` examples.

**Reply by HartS** [reply]
@bhushankumar-developer @sumitalp I was seeing the same issue as well (using node-js example). It turns out I needed to set `SECURE_CHANNELS=false` and remove the `JWT_SECRET` env from the realtime server in order to get it working from the example

**Reply by HartS** [reply]
Of course, once you're beyond the example, you probably *do* want those channels to be secured with JWT

**Reply by soedirgo** [reply]
Thanks @HartS, both the next-js and node-js examples are updated now.
